app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class SessionsController < ApplicationController
+  def create
+    begin
+      if user = User.from_omniauth(request.env['omniauth.auth'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.